examples/context-based-restrictions.v1.test.js

```diff
@@ -0,0 +1,463 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-console */
+
+const ContextBasedRestrictionsV1 = require('../dist/context-based-restrictions/v1');
+// eslint-disable-next-line node/no-unpublished-require
+const authHelper = require('../test/resources/auth-helper.js');
+// You can use the readExternalSources method to access additional configuration values
+const { readExternalSources } = require('ibm-cloud-sdk-core');
+
+//
+// This file provides an example of how to use the Context Based Restrictions service.
+//
+// The following configuration properties are assumed to be defined:
+// CONTEXT_BASED_RESTRICTIONS_URL=<service base url>
+// CONTEXT_BASED_RESTRICTIONS_AUTH_TYPE=iam
+// CONTEXT_BASED_RESTRICTIONS_APIKEY=<IAM apikey>
+// CONTEXT_BASED_RESTRICTIONS_AUTH_URL=<IAM token service base URL - omit this if using the production environment>
+// CONTEXT_BASED_RESTRICTIONS_TEST_ACCOUNT_ID=<the id of the account under which test CBR zones and rules are created>
+// CONTEXT_BASED_RESTRICTIONS_TEST_SERVICE_NAME=<the name of the service to be associated with the test CBR rules>
+//
+// These configuration properties can be exported as environment variables, or stored
+// in a configuration file and then:
+// export IBM_CREDENTIALS_FILE=<name of configuration file>
+//
+const configFile = 'context_based_restrictions_v1.env';
+
+const describe = authHelper.prepareTests(configFile);
+
+const originalLog = console.log;
+
+const consoleLogMock = jest.spyOn(console, 'log');
+const consoleWarnMock = jest.spyOn(console, 'warn');
+
+describe('ContextBasedRestrictionsV1', () => {
+  const contextBasedRestrictionsService = ContextBasedRestrictionsV1.newInstance({});
+  const config = readExternalSources(ContextBasedRestrictionsV1.DEFAULT_SERVICE_NAME);
+  const {
+    apikey: apiKey,
+    url: URL,
+    authUrl,
+    authType,
+    testAccountId: accountId,
+    testServiceName: serviceName,
+  } = config;
+
+  expect(config).not.toBeNull();
+  expect(accountId).not.toBeNull();
+  expect(accountId).toBeDefined();
+  expect(apiKey).not.toBeNull();
+  expect(apiKey).toBeDefined();
+  expect(URL).not.toBeNull();
+  expect(URL).toBeDefined();
+  expect(authUrl).not.toBeNull();
+  expect(authUrl).toBeDefined();
+  expect(authType).not.toBeNull();
+  expect(authType).toBeDefined();
+  expect(serviceName).not.toBeNull();
+  expect(serviceName).toBeDefined();
+
+  let zoneId;
+  let zoneRev;
+  let ruleId;
+  let ruleRev;
+
+  test('createZone request example', async (done) => {
+    consoleLogMock.mockImplementation((output) => {
+      originalLog(output);
+      done();
+    });
+    consoleWarnMock.mockImplementation((output) => {
+      done(output);
+    });
+
+    originalLog('createZone() result:');
+
+    const addressModel = {
```

Comment:
```suggestion
    // begin-create_zone
    
    const addressModel = {
```
these begin/end comments will need to be added back to each operation's example code fragment

Reply:
added comment back